keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import warnings
+
+import tensorflow as tf
+
+from keras_cv.layers import BaseImageAugmentationLayer
+from keras_cv.utils import preprocessing
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class RandomResizedCrop(BaseImageAugmentationLayer):
+    """
+    Randomly crops a part of an image and resizes it to provided size.
```

Comment:
Should we briefly jot down the approach that we're taking here. There are many implementation of RandomResizedCrop. I believe if we jot down our summary here it'd be easier for the community to understand what we're doing.

Reply:
Sure. The basic approach in the current implementation is same as one would expect: randomly crop the images and resize to given target size. I'll add a line regarding that.

Reply:
I think it would be beneficial to also expand a little bit on our approach of randomly cropping an image in the first place. Some implementations sample lower and upper bounds on the coordinates which are then used for cropping. But it varies. Hence, I am suggesting it.